src/routes/index.js

```diff
@@ -0,0 +1,13 @@
+import user from './UserRoutes';
+import Res from '../utils/helpers/responses';
+
+const prefix = '/api/v1';
+
+const router = (app) => {
+    app.use(prefix, user);
+    // Res.handleError below acts as success
+    app.get('/', (req, res) => Res.handleError(200, 'Welcome to Property Pro Lite', res));
+    app.use('*', (req, res) => Res.handleError(405, 'Invalid route or Method', res));
```

Comment:
This shouldn't be 405, let's have a 2 minutes sync and show you why.

Reply:
Fixed it, handled both the 405 and the 404
Thanks for that